packages/client/src/components/QuestionResult/QuestionResult.styles.ts

```diff
@@ -28,15 +27,12 @@ export const QuestionResultDiv = styled.div`
 `;
 
 export const IconBorder = styled.div<QuestionResultProps>`
-    width: 4.0rem;
-    height: 4.0rem;
+    width: ${t.sizes.xl};
+    height: ${t.sizes.xl};
```

Comment:
Good for now. Keep in mind @wdaimee we also have the `t.size('XYZ')` function! 😄